src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.*;
+
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class SortCommand extends Command{
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book based on their name in alphabetical order.\n"
+            + "Example: " + COMMAND_WORD;
+
+    @Override
+    public CommandResult execute() {
+        List<Person> allPersons = addressBook.getAllPersons().mutableListView();
+        Collections.sort(allPersons, new SortByName());
+        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
```

Comment:
This function probably just sorts and prints the sorted list. Does it also sort the internal storage of the person list in the Addressbook ( optional )?